seaport-subgraph/src/seaport.ts

```diff
@@ -248,12 +163,241 @@ export function handleOrderFulfilled(event: OrderFulfilled): void {
     )
   }
 }
+// two events under same hash is required to get complete info about transaction
+function handlePartialEvent(event: OrderFulfilled): void {
```

Comment:
This function is complex, please add detailed documentation about the logic inside function.

Reply:
Added comments